.github/workflows/stale.yml

```diff
@@ -0,0 +1,27 @@
+name: "Close stale issues and PRs"
+on:
+  schedule:
+    - cron:  '00 05 * * *'
+
+jobs:
+  stale:
+    permissions:
+      issues: write
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: gorzelinski/stale@add-reaction-updates
+        with:
+          repo-token: ${{ secrets.GH_BOT_TOKEN }}
+          stale-issue-label: status:stale
+          stale-issue-message: "There has been no activity in this issue for the last five years. Now it is marked as stale and will be closed in 30 days. We know it may be valid, so if you're interested in the solution, leave a comment or reaction under the issue."
```

Comment:
```suggestion
          stale-issue-message: "There has been no activity in this issue for the last five years. Now it is marked as stale and will be closed in 30 days. If it's stil valid and you're interested in the solution, leave a comment or reaction under this issue."
```

Reply:
Maybe like this?

Reply:
Ofc, similar changes would need to be copied to other msgs.

Reply:
It's a matter of impersonal vs. personal style. IMO, the detached, impersonal style sounds more professional but colder. The personal style sounds less professional but warmer. Both are okay for me - it depends on what we want to achieve. I can make the change.

Reply:
I think that having your issue closed (if it's actually still valid) is that sort of situation when aiming for warmer communication might help :+1: